python/ray/data/read_api.py

```diff
@@ -468,6 +468,7 @@ def read_text(
     *,
     encoding: str = "utf-8",
     errors: str = "ignore",
+    filter_empty_lines: bool = True,
```

Comment:
How about calling it drop_empty_lines?
```suggestion
    drop_empty_lines: bool = True,
```